packages/client/src/v2-events/utils/commonUtils.ts

```diff
@@ -0,0 +1,41 @@
+/*
```

Comment:
Let's import this directly from the legacy side if there are no changes needed for this